py/server/deephaven/table.py

```diff
@@ -1271,46 +1270,38 @@ def aj(self, table: Table, on: Union[str, Sequence[str]], joins: Union[str, Sequ
 
         Args:
             table (Table): the right-table of the join
-            on (Union[str, Sequence[str]]): the column(s) to match, can be a common name or an equal expression,
-                i.e. "col_a = col_b" for different column names
+            on (Union[str, Sequence[str]]): the column(s) to match, can be a common name or a match condition of
+                two columns, e.g. 'col_a < col_b'. The other valid comparison operator is '<='.
```

Comment:
Only the _last_ column can have a non-equals comparison operator:

```python
# these are all less-than-or-equal on baz
["foo", "bar", "baz"]
["foo", "bar", "baz=baz"]
["foo", "bar", "baz==baz"]
["foo", "bar", "baz<=baz"]

# this is the only logically different one
["foo", "bar", "baz<baz"]
```

Reply:
I agree that what is here is not clear.

Reply:
```suggestion
            on (Union[str, Sequence[str]]): the column(s) to match, can be a common name or a match condition of
                two columns, e.g. 'col_a = col_b'. The first 'N-1' matches are exact matches.  The final match is an inexact match.  The inexact match can use either '<' or '<='.  If a common name is used for the inexact match, '<=' is used for the comparison.
```